routing/routing_session.hpp

```diff
@@ -34,13 +34,35 @@ class RouteMatchingInfo;
 
 namespace routing
 {
-struct SpeedCameraRestriction
+struct SpeedCameraOnRoute
 {
-  size_t m_index;  // Index of a polyline point where camera is located.
-  uint8_t m_maxSpeedKmH;  // Maximum speed allowed by the camera.
+  SpeedCameraOnRoute() = default;
+  SpeedCameraOnRoute(double distFromBegin, uint8_t maxSpeedKmH)
+    : m_distFromBegin(distFromBegin), m_maxSpeedKmH(maxSpeedKmH)
+  {}
+
+  static double constexpr kAverageAccelerationOfBraking = -6.4697;  // Meters to square seconds.
```

Comment:
static_assert that |kAverageAccelerationOfBraking| != 0?

Reply:
done